web/src/i18n/locales/en.yaml

```diff
@@ -172,18 +172,18 @@
     description: 
       title: "Activity Overview"
       helpText: "Summarize the needs of the activity for which requested funding will be used (e.g. administrative, outreach, auditing, Public Health Registry Development, etc)"
-      reminder: "This should be very brief."
+      reminder: "The activity overview should be a brief summary of the activity needs."
       summary: 
         subheader: "Brief Activity Summary"
       detail:
         standard:
-          subheader: "Describe the activity and include relevant information."
+          subheader: "The activity summary should be brief, there will be additional opportunities to expand in other sections."
```

Comment:
This section below the brief activity summary doesn't have to be brief. This should be as long as needs to be in order to explain what they are doing. Are we missing a title for this text box?

![image](https://user-images.githubusercontent.com/34279968/42399618-6370c686-8123-11e8-8181-691deb919265.png)